### PR TITLE
Web client

### PR DESCRIPTION
**Pull request by dmdashenkov**
This PR introduces the `web` module.

This module is a simple example of how to expose a Spine application over HTTP and how to create a JavaScript client for such an application.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

File: web/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Note that the client module is a Gradle project. Because of this, the client is capable of compiling
+JS sources from the Protobuf definitions. See [the build script](./client/build.gradle) for 
+the details.
+
+## Running
+
+In order to contemplate the web modules in action, follow these steps:
+ 1. In the `web/client`:
+    - assemble the JavaScript via `npm install`;
+    - build a single javascript artifact via `webpack`.
```
Comment (armiol) [inline web/README.md]: `JavaScript`.

File: web/server/src/main/java/io/spine/quickstart/web/package-info.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+@CheckReturnValue
+@ParametersAreNonnullByDefault
+package io.spine.quickstart.web;
```
Comment (armiol) [inline web/server/src/main/java/io/spine/quickstart/web/package-info.java:30]: Please describe the package.

File: web/server/src/main/java/io/spine/quickstart/web/TasksCommandServlet.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+package io.spine.quickstart.web;
+
+import io.spine.quickstart.TasksContext;
+import io.spine.web.command.CommandServlet;
+
+import javax.servlet.annotation.WebServlet;
+
+@WebServlet("/command")
+public final class TasksCommandServlet extends CommandServlet {
```
Comment (armiol) [inline web/server/src/main/java/io/spine/quickstart/web/TasksCommandServlet.java:38]: Please have Javadocs for this and other servlets.

File: web/client/src/main/index.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+import * as firebase from 'firebase/app';
+import 'firebase/database';
+
+const HOST = 'http://localhost:8080';
+const FIREBASE = firebase.initializeApp({
+    databaseURL: 'ws://localhost:5000/',
+    authDomain: 'ws://localhost:5000/'
+});
+
+class TaskController {
```
Comment (armiol) [inline web/client/src/main/index.js:46]: Please document this class and two functions below.

**Review by armiol — CHANGES_REQUESTED** [review]
@dmdashenkov please see my comments.

**Comment by dmdashenkov** [comment]
@armiol, PTAL again.

**Comment by dmdashenkov** [comment]
@armiol, PTAL again.

File: web/client/build.gradle [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+compileJava.enabled = false
+compileTestJava.enabled = false
+
+configurations.protobuf {
+    transitive = false
+}
+
+protobuf {
```
Comment (armiol) [inline web/client/build.gradle:34]: Let's create a task to include the configuration like this one and the one above into the Bootstrap plugin.

**Review by armiol — APPROVED** [review]

**Pull request merged**